src/main/java/org/sonar/plugins/cas/session/SessionStoreCleaner.java

```diff
@@ -0,0 +1,78 @@
+package org.sonar.plugins.cas.session;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.sonar.api.config.Configuration;
+import org.sonar.api.platform.Server;
+import org.sonar.api.platform.ServerStartHandler;
+import org.sonar.api.server.ServerSide;
+import org.sonar.plugins.cas.util.SonarCasProperties;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+@ServerSide
+public final class SessionStoreCleaner implements ServerStartHandler {
+    private static final Logger LOG = LoggerFactory.getLogger(SessionStoreCleaner.class);
+    private static final int SESSION_STORE_CLEANUP_INTERVAL_IN_SECS_DEFAULT = 60 * 30;
```

Comment:
We could use `TimeUnit` here, it should be more readable.

Reply:
`TimeUnit.MINUTES.toSeconds(30l)`

Reply:
resolved